src/components/HighestLeaders/HighestLeaders.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+// IMG
+import avatar from '../../img/avatar.svg';
+import img from '../../img/business-people.svg';
+// SCSS
+import './HighestLeaders.scss';
+import { useSelector } from 'react-redux';
+import getLeaders from '../../store/appStores/leaderStore/selector';
+
+const HighestLeaders = () => {
+  const users = useSelector(getLeaders);
+  console.log('users', users);
+
+  return (
+    <div className="leaders">
+      <div className="leaders_container">
+        <div className="leaders_box">
+          <h3>All time Highest Scorers</h3>
+          <div>
+            <ul className="leaders_list list">
+              {/* {users?.map((user: any) => (*/}
```

Comment:
If you comment some of the code because it's going to be implemented in future PR, please, add `TODO: ` prefix.